project/RELEASE-VERSIONING.md

```diff
@@ -0,0 +1,28 @@
+# VIC Release Versioning
+
+This document provides examples of VIC release versions.
+
+The VIC project follows the general vX.Y.Z versioning scheme. Where X is Major, Y is
```

Comment:
Just realized this is 3 separate files, not sections.. any reason not to have these in a single file?